README.md

```diff
@@ -113,3 +113,28 @@ schema-tools, make sure one of the commit increments the version number in
 * tag the release X.Y.Z with ``git tag -a vX.Y.Z -m "Bump to vX.Y.Z"``;
 * push the tag to GitHub with ``git push origin --tags``;
 * release to PyPI: ``make upload`` (requires the PyPI secret).
+
+
+## Mocking data
+
+The schematools library contains two Django management commands to generate
+mock data. The first one is `create_mock_data` which generates mock data for all
+the datasets that are found at the configured schema location `SCHEMA_URL`.
+
+The `create_mock_data` command has some options, e.g. to change
+the default number of generated records (`--size`), to skip datasets (`--skip`)
+or to use a subset of the datasets (`--limit-to`). To avoid duplicate pk's
+on subsequent runs the `--start-at` options can be used.
```

Comment:
I'm not sure why we need two options to control the datasets. Why not have

    schema create_mock_data

generate mock data for all datasets, and have

    schema create_mock_data foo bar

generate data only for `foo` and `bar`? Right now, I can't even figure out how to use `--limit-to`. Is it

    schema create_mock_data --limit-to="foo bar"
    schema create_mock_data --limit-to=foo,bar
    schema create_mock_data --limit-to=foo --limit-to=bar

?

When the datasets are ordinary cmdline args, `--skip` can just be a boolean flag and inconsistent combinations like `--limit-to=foo --skip=foo` become impossible.

Reply:
Good point, that's a better cli design. However, `--skip` should not be a boolean, but one or more dataset ids. So that one can create mocks for all dataset but exclude just a few (usually the proxied datasets, because these datasets do not need database tables).  Thinking about it, we have the flag `enable_db` in `datasets_dataset`, so it is even more convenient to always exclude these datasets. I'll implement that.

Reply:
The reason for introducing `--limit-to` was that the positional arguments can be one or more paths to dataset schema files. However, it would be easy to find out if an argument is a local path or just a dataset identifier.

Reply:
If the datasets are positional args, then we can say

    schema create_mock_data --skip foo bar baz

to process everything but foo, bar and baz.

And don't we already have code to handle `--schema-url` specially for local paths for other commands?

Reply:
The code that uses `--schema-url` is not being used in the Django based mgm. commands (like `import_schemas`). I now applied the same behaviour that `import_schemas` has to these new mgm. commands.
I'll have a look if the `--schema-url` based approach can also be applied to the Django mgm. commands.

Reply:
It was not too hard to integrate the same schemaloader for the Django mgm. commands, so I added it.

Reply:
And also added this to the `import_schemas` Django mgm. command.

Reply:
Exclude datasets with an `enable_db` is False.